### ISSUE TITLE
7ktTube greyed out clicked thumbnails

### ISSUE THREAD

**Issue by K4sum1** [lightbeam24/CustomTube#100]
One of the features of 7ktTube that I really like is greyed out (and blurred) thumbnails after clicking, It easily tells me if I already saw a video. I would like this feature (and importing of history data from 7ktTube) to be added to CustomTube.

**Comment by lightbeam24** [comment]
I was actually thinking of implementing something like this, except it wouldn't blur or grey out the thumbnails. Maybe I could look into adding an option to grey them out as well.

> (and importing of history data from 7ktTube)

This won't be done. I don't know how I would achieve something like that. I would have to learn how 7ktTube does it (Also, why would it need to store its own watch history? That makes no sense to me). Rather, I have a CSS-based solution based off of your YouTube watch history:

Any video which has a red progress bar on the thumbnail:
![Screenshot_2023-10-16_09-02-24](https://github.com/lightbeam24/CustomTube/assets/105595945/6dc04256-f760-4a48-9ca5-4c4444b1b03a)
Gets a watched overlay:
![Screenshot_2023-10-16_09-02-36](https://github.com/lightbeam24/CustomTube/assets/105595945/e688d7b9-82cd-4452-ad80-e09088c05afb)
The same thing could theoretically be done with blurring and greying out as well.

**Comment by lightbeam24** [comment]
Inconsistent how? YouTube's red bar doesn't lie in my experience.

**Comment by K4sum1** [comment]
I've seen plenty of greyed out videos that I know I have watched with no bar.

**Comment by lightbeam24** [comment]
Have you switched YouTube accounts at all and/or are using multiple accounts? Either that or your Google account must be set to auto delete YouTube watch history after a set amount of time (3 months possibly). Or are neither of those the case and it's really just inconsistent?

**Comment by K4sum1** [comment]
No, I have it all in a separate browser I just watch YouTube in, and I don't touch anything. I have the red bar under pretty old videos too.

**Comment by lightbeam24** [comment]
Oh, well sorry, but I really don't think I'm going to do it the way 7ktTube does it. I wouldn't even know where to begin.
